apps/data-types.md

```diff
@@ -83,15 +111,11 @@ This object is used for the attributes of shippingAddress and billingAddress.
 | internalNote | string | Internal note for the customer done by the merchant. |
 | links | array of [link](page:apps-data-types#link) | The link to the customer and their orders. |
 
-## customAttribute
+## customer (create request)
 
 | Attribute | Type | Description |
 | - | :-: |  - |
-| key | string | The identifier of the custom attribute. |
-| displayKey | string | The displayed name of the custom attribute. |
-| singleValue | boolean | Indicates if just one feature is selected for the custom attribute.  |
-| type | enum | The data type of the custom attribute. Can be *string*, *number*, *bool*, *datetime*, *time*, *url*.|
-| values | array of [variationValue](page:apps-data-types#variationvalue)| The options selected for the custom attribute. |
+| customerNumber | string | The number by which the customer is tracked.|
```

Comment:
mention that customerNumber is optional. If not provided, EP6 will generate one.